Formula/bgfx.rb

```diff
@@ -0,0 +1,62 @@
+class Bgfx < Formula
+  desc "Cross-platform, graphics API agnostic, rendering library"
+  homepage "https://github.com/bkaradzic/bgfx"
+  url "https://github.com/bkaradzic/bgfx.cmake.git",
```

Comment:
These seem to be two different projects, with the latter only being as a C++ library, and it seems you want to package the former.

Reply:
My understanding is that the C++ library's build system lives in a separate (forked) repo, and that's the source we're using here.

Reply:
@carlocab, correct. I'm sorry this formula is turning out to be so difficult. 

The `bgfx.cmake` repo was started by someone else, but then archived, and brought in to github.com/bkaradzic to be official. The problem is it wasn't transferred, so it still has the fork status. I've read that you can request github to remove that flag. 

I've reached out to bkaradzic a few times, however I am still waiting for a response.